meerk40t/core/elements.py

```diff
@@ -64,6 +66,17 @@ def plugin(kernel, lifecycle=None):
 
 MILS_IN_MM = 39.3701
 
+group_simplify_re = re.compile("(\([^()]+?\))|(SVG(?=Image|Text))|(Simple(?=Line))", re.IGNORECASE)
+element_simplify_re = re.compile("(^Simple(?=Line))|((?<=\.\d{2})(\d+))", re.IGNORECASE)
+image_simplify_re = re.compile("(^SVG(?=Image))|((,\s*)?href='data:.*?'(,\s?|\s|(?=\))))", re.IGNORECASE)
+
+OP_PRIORITIES = ["Dots","Image","Raster","Engrave","Cut"]
+
+def reversed_enumerate(collection: list):
```

Comment:
It is indeed functionally the same but I think that `reversed(list(enumerate(c)))` is an iterator on `list(enumerate(c))` which is created rather than being a reversed iterator on `c` itself. I doubt that the operations list will ever be long enough for this to be an issue - happy to switch back if you prefer.

Reply:
Agreed in IRC to leave this as-is.